src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetBuilding.java

```diff
@@ -317,7 +289,7 @@ private boolean placeBlock(World world, EntityPlayer player, BlockPos pos, IBloc
             useConstructionPaste = true;
         }
 
-        if( !this.canUse(heldItem, player) )
+        if (!this.canUse(heldItem, player))
```

Comment:
There wasn't anything wrong with the original?

Reply:
What do you mean?

Reply:
Spaces were fine?

Reply:
There should always be a space after the `if`, and never spaces between the statement's parenthesis and the code they enclose.

Reply:
Not a fan but alright